Taggers/interface/CollectionDumper.h

```diff
@@ -184,6 +186,9 @@ namespace flashgg {
         if( dumpWorkspace_ ) {
             ws_ = fs.make<RooWorkspace>( workspaceName_.c_str(), workspaceName_.c_str() );
             dynamic_cast<RooRealVar *>( ws_->factory( "weight[1.]" ) )->setConstant( false );
+            RooRealVar* intLumi = new RooRealVar("IntLumi","IntLumi",intLumi_*0.001,0,30000000);
```

Comment:
Please make this variable constant, as we don't want it to be floated in the fits. (

The downstream scripts can always enforce the constantness, but better to start on the right foot.

(also, you are not required to specify a range).

Btw, I see that for some reason the weight is not set constant and I am not sure why.